src/TestRotations.java

```diff
@@ -0,0 +1,55 @@
+//: object/TestRotations.java
+
+import junit.framework.*;
+
+/**
+ * JUnit Class to test the rotation methods from the Quaternion and RotationMatrix classes.
+ *
```

Comment:
That's a great question! The answer is that it depends; are the edge cases the same for both tests? So e.g. an edge case for the rotation matrix may be trying a matrix that has all zeroes, but that may not have sense for a quaternion. So  you have to always think what are the edge cases for the class that you are writing the tests.